tests/envs/test_cover.py

```diff
@@ -339,22 +314,32 @@ def test_cover_multistep_options():
         # Ungrasp
         np.array([0., 0., -0.1], dtype=np.float32),
     ]
-    make_video = False  # Can toggle to true for debugging
 
-    def policy(s: State) -> Action:
-        del s  # unused
-        return Action(action_arrs.pop(0))
+    policy = utils.action_arrs_to_policy(action_arrs)
+
+    # Here's an example of how to make a video within this test.
+    # env.reset("train", 0)
```

Comment:
Why is this needed? (I think I know the answer, and it's related to the difference between `env.render` and `env.render_state`...)

Reply:
yeah right. since the video monitor uses render, and since render relies on the current state and current task in the env being correct, we need to reset the environment. a bit unfortunate, but since this is only for tests I think okay